### PR TITLE
Hide team display in leaderboards for a particular tournament

### PR DESCRIPTION
**Pull request by nwinter**
(tournament owner doesn't want teams showing up here)

Test: /play/ladder/desert-duel/clan/65775bd45f6f1900185abf5b?tournament=65775da26dd00500194eaf3f

### CODE REVIEW

File: app/views/ladder/components/Leaderboard.js [hunk 1]
```diff
@@ -51,6 +54,9 @@ module.exports = (LeaderboardView = (function () {
         { slug: 'age', col: 1, title: $.i18n.t('ladder.age_bracket') },
         { slug: 'country', col: 1, title: '🏴‍☠️' }
       ]
+      if (this.hidesTeams) {
+        this.tableTitles.splice(7, 1)
```
Comment (smallst) [inline app/views/ladder/components/Leaderboard.js:58]: 7 is fine, but maybe using `_.find {slug:'clan'}` is better?

File: app/views/ladder/components/Leaderboard.js [hunk 2]
```diff
@@ -64,6 +70,9 @@ module.exports = (LeaderboardView = (function () {
           { slug: 'when', col: 1, title: $.i18n.t('general.when') },
           { slug: 'fight', col: 1, title: '' }
         ]
+        if (this.hidesTeams) {
+          this.propsData.tableTitles.splice(5, 1)
```
Comment (smallst) [inline app/views/ladder/components/Leaderboard.js:74]: same here. otherwise 7 vs 5 maybe confused in the future

**Review by smallst — APPROVED** [review]
LGTM. left unblocking comments

File: app/views/ladder/components/Leaderboard.js (hunk 2 above)
Comment (mrfinch) [reply app/views/ladder/components/Leaderboard.js:74]: we should not use magic-numbers like 5. Better to at least give them a variable name

**Pull request merged**